packages/workers-shared/asset-server-worker/src/index.ts

```diff
@@ -1,5 +1,4 @@
 import { AssetsManifest } from "./assets-manifest";
-import { NoopAssetsManifest } from "./assets-manifest-no-op";
 
 interface Env {
 	ASSETS_MANIFEST: ArrayBuffer;
```

Comment:
Consider putting the documentation comments on here (as JSDoc) rather than on the const variable definitions inside the fetch handler.